API.md

```diff
@@ -484,6 +586,16 @@ The key-value pairs of this object are described by the telemetry metadata of
 a domain object, as discussed in the [Telemetry Metadata](#telemetry-metadata)
 section.
 
+#### Limit Evaluators **draft**
+
+Limit evaluators allow a telemetry integrator to define how limits should be 
+applied to telemetry from a given domain object.  For an example of a limit 
+evaluator, take a look at `examples/generator/SinewaveLimitProvider.js`.
+
+### Telemetry Visualization APIs **draft**
+
+The APIs for fetching and visualizing telemetry-- those that would be used in custom views-- are currently in draft state and are being revised.  If you'd like to experiement with them before they are finalized, please contact the  team via the contact-us link on our website.
```

Comment:
I wonder if perhaps we should provide at least some rudimentary API docs here, with the big caveat that they are draft. We are building views with the new API now, so I think our basic approach is fixed, even though some details might change. It could be as simple as https://github.com/nasa/openmct/issues/1807#issuecomment-344345782

Reply:
clarification: andrew refers to "view" api, not telemetry accessors.

Reply:
Will clarify this segment to discuss telemetry retrieval api so that it isn't confused with View API.